index.html

```diff
@@ -1 +1,106 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
+
+<p>The Email Markup Consortium (EMC) is a community-led group of industry professionals working to improve the email experience for everyone.</p>
+
+<p>Senders, recipients, and email clients share the common goals of improving:</p>
+
+<ol>
+  <li>user experience</li>
+  <li>accessibility</li>
+  <li>performance</li>
+  <li>consistency</li>
+  <li>reliability</li>
+</ol>
+
+<p>However, currently there is a lack of cooperation in trying to achieve these goals leading to inconsistent results. By working together on agreed approaches we can save everyone time, effort, and money while delivering better results to the recipients.</p>
+
+<p>Recipients should be able to easily access the content of every email sent to them, regardless of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation and sending tools, and email clients to drive this vision forward at every step of the email journey.</p>
+
+
+<h2>Quick Links</h2>
+
+<ul>
+  <li>
+    <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/mission.md">
+      Mission statement
+    </a>
+  </li>
+  <li>
+    <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/code-of-conduct.md">
+      Code of conduct
+    </a>
+  </li>
+  <li>
+    <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/compliant-standards.md">
+      Compliant Standards
+    </a>
+  </li>
+  <li>
+    <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/glossary.md">
+      Glossary
+    </a>
+  </li>
+</ul>
+
+
+<h2>How You Can Help</h2>
+
+<h3>Join Us</h3>
+<ul>
+  <li><a href="https://forms.gle/XVdT3JGUbLZQEHXu6">Individual membership application form</a></li>
+  <li><a href="https://forms.gle/yifokDd3rfdFRG7i8">Corporate membership application form</a></li>
+  <li>Not ready to apply? Join the <a href="https://github.com/email-markup-consortium/email-markup-consortium/discussions">discussion on GitHub</a></li>
+</ul>
+
+
+<h3>Spread the Word</h3>
+<ul>
+  <li><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Femailmarkup.org">Twitter</a></li>
+  <li><a href="https://www.linkedin.com/shareArticle?mini=true&url=https%3A%2F%2Femailmarkup.org">LinkedIn</a></li>
+  <li><a href="https://www.facebook.com/sharer?u=https%3A%2F%2Femailmarkup.org">Facebook</a></li>
+</ul>
+
+<h2>Supporters</h2>
+
+<p>The Email Markup Consortium and its mission are supported by employees of Salesforce, Squarespace, Litmus, Email on Acid, SparkPost, Constant Contact, ActionRocket, Parcel, and more.</p>
```

Comment:
Could it be good to include something about locations of members? 
"With members spanning across the globe"
"With members on [x] continents"
"With members in [x] [y] [z]"

I'm not sure where everyone is based, so this might not look so impressive, but also that could show gaps in our coverage, maybe we should look for members in areas we don't yet have people.

Reply:
As far as I know we have members in the UK, the US, Europe and the middle east (✋). Not sure if some of us are in other regions, but that's not a bad mix to start with.

Reply:
We could probably just use "worldwide" or "globally" or something.

Let's figure out how to get that line above the Contributors list in a follow-up. See https://github.com/email-markup-consortium/email-markup-consortium.github.io/issues/4.